packages/aws-cdk/lib/api/toolkit-info.ts

```diff
@@ -143,6 +143,14 @@ export class ToolkitInfo {
       lifecyclePolicyText: JSON.stringify(DEFAULT_REPO_LIFECYCLE)
     }).promise();
 
+    // Activate image scanning on push (helps in identifying software vulnerabilities, no additional charge)
```

Comment:
feels like this should be configurable, no?

Reply:
It adds security and is free, why?

Reply:
Let me ask it this way, could this block the push? If yes, we must allow users to opt-out.

Reply:
It's a asynchronous process and then 
> Amazon ECR sends an event to Amazon EventBridge (formerly called CloudWatch Events) when an image scan is completed.

Reply:
ok cool. so makes sense